docs/releases/release_notes.md

```diff
@@ -32,11 +32,7 @@ cf version
 ### Mac
 ```bash
 # download and extract the binary
-curl -L --output - https://github.com/codefresh-io/cli-v2/releases/download/v0.0.109/cf-darwin-amd64.tar.gz | tar zx
-
-# move the binary to your $PATH
-mv ./cf-darwin-amd64 /usr/local/bin/cf
-
-# check the installation
-cf version
-```
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
will make sure none of those remain and that all is valid after `make codegen` (which would be possible after merging go-sdk)